js/google_ima.js

```diff
@@ -937,7 +937,10 @@ require("../html5-common/js/utils/utils.js");
 
           //iphone performance is terrible if we don't use the custom playback (i.e. filling in the second param for adDisplayContainer)
           //also doesn't not seem to work nicely with podded ads if you don't use it.
-          _IMAAdDisplayContainer = new google.ima.AdDisplayContainer(_amc.ui.adWrapper[0],
+
+          //for IMA, we always want to use the plugins element to house the IMA UI. This allows it to behave
+          //properly with the Alice skin.
+          _IMAAdDisplayContainer = new google.ima.AdDisplayContainer(_amc.ui.pluginsElement[0],
```

Comment:
IMA is special in that it controls the ad UI and provides the capability for us to provide to them a container to store this UI and a container to place the video. I believe we should only use the plugins element (instead of the ad wrapper) for situations like these. Our other ad managers behave differently from IMA and should still use ad wrapper to house the video. We control the UI for our other ad managers, at least the current ones.